lib/sla.bootstrap.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var util = require('util');
+
+var ari = require('ari-client');
+var Q = require('q');
+
+var sla = require('./sla.application.js');
+var config = require('./config/app.config.json');
+
+/**
+ * Waits for a StasisStart event before going into the main SLA module.
+ * @param {Object} client - Object that contains information from the ARI
+ *   connection.
+ * @param {String} confFilePath - File path for the configuration file.
+ */
+function clientLoaded (client, confFilePath) {
+  client.start('sla');
+
+  client.on('StasisStart', function(event, channel) {
+    var extension = event.args[0];
+
+    // Channels that we have dialed from within an SLA instance should not
+    // spin up a new instance of the application.
+    if (extension !== 'dialed') {
+      var app = sla(client, confFilePath, channel, extension);
+
+      app.run();
+    }
+  });
+}
+
+/**
+ * Creates the ari connection and then allows the StasisStart listener to be
```

Comment:
This is strangely worded. Just say "and then registers a StasisStart event handler" instead of saying "and then allows..."